woocommerce/woocommerce
# [Store Alerts] Adjust inner styles

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/47034

This PR updates the Store Alert styles.

Before
<img width="739" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/e07383d8-8b15-433d-8b94-d4bf82ddbd6c">


After
<img width="717" alt="image" src="https://github.com/woocommerce/woocommerce/assets/1922453/df7f6657-d2f8-49f2-9cae-2ed90b964a5e">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Make sure you have more than one Store Alert visible

1. Trigger multiple notes. First see what's already in your DB.
```sql
SELECT note_id,is_read,is_deleted,status,type from wp_wc_admin_notes WHERE type = 'update'
```
2. Make a few "unactioned".
```sql
UPDATE wp_wc_admin_notes SET status = 'unactioned' WHERE note_id IN (1,2,3,4);
```
3. You may need to mark some as not deleted.
```sql 
UPDATE wp_wc_admin_notes SET is_deleted = 0 WHERE note_id IN (1,2,3,4);
```
4. See alerts on various screens like Home, Customers, Settings and be sure to check various sizes.
5. Compare to designs found at oDTJrNy6IXwdgFNinZmYgI-fi

<img width="822" alt="Main component" src="https://github.com/woocommerce/woocommerce/assets/5121465/cc0b9881-7469-471f-add8-ef53d8f72c52">



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Update Store Alert styles

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
